.github/workflows/ci.yaml

```diff
@@ -44,7 +44,12 @@ jobs:
           sudo pip3 install yamllint
 
       - name: Run linters
-        run: ./hack/pre-commit.sh
+        run: |
+           ./hack/pre-commit.sh
+           echo "Checking that generated files were not modified"
+           make generate
```

Comment:
I think we need `go mod tidy` as well here, to detect changes to go.mod and go.sum files that were not part of the PR.

In addition `make manifests` maybe required to ensure generated YAML files are part of the PR